src/services/role/services/rolesService.js

```diff
@@ -0,0 +1,102 @@
+const { NotFound } = require('@feathersjs/errors');
+const sift = require('sift');
+const { RoleModel } = require('../model');
+const { preparedRoles, unique } = require('../utils/preparedRoles');
+
+// Do not proteced this route with authentication.
+const RoleServiceHooks = {
+	before: {
+		all: [],
+		find: [],
+		get: [],
+	},
+};
+
+const filterByQuery = (roles = [], query = {}) => {
+	let result = roles;
+	const q = Object.assign({}, query);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use an object spread instead of `Object.assign` eg: `{ ...foo }`.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5258109)

Reply:
bäh immer dieser linter bot. :P

Array (Iterable) Spread Example

const x = [1, 2, null , 3];
const y = [...x, 4, 5];
const z = null;

console.log(y); // [1, 2, null, 3, 4, 5];
console.log([...z]); // TypeError

Object Spread Example

const x = null;
const y = {a: 1, b: 2};
const z = {...x, ...y};

console.log(z); //{a: 1, b: 2}

Das macht schon ein unterschied...